### PR TITLE
DT-1655 new metadata

### PR DESCRIPTION
**Pull request by Macroz**
The JIRA-issue lists a bunch of open questions but otherwise review can be started already.

Image has not been updated yet.

### CODE REVIEW

File: app/config.hsl.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const CONFIG = process.env.CONFIG || 'hsl';
 const API_URL = process.env.API_URL || 'https://dev-api.digitransit.fi';
-const APP_DESCRIPTION = 'HSL:n uuden Reittioppaan kokeiluversio.';
+const APP_DESCRIPTION = 'Helsingin seudun liikenteen uusi Reittiopas.';
```
Comment (hannesj) [inline app/config.hsl.js]: Where does this text come from?

Comment (Macroz) [inline app/config.hsl.js]: From slack discussion. It's obviously time to remove the "kokeiluversio" and this just essentially says the same.

File: app/component/StopPageMeta.js [hunk 2]
```diff
@@ -7,20 +7,37 @@ import { intlShape } from 'react-intl';
...
-      content: props.intl.formatMessage({
-        id: 'stop-page.description',
-        defaultMessage: 'Stop {stop_name} - {stop_code}',
-      }, { stop_name: props.stop.name, stop_code: props.stop.code }),
-    }],
-  })),
+        params.stopId ? 'Stop - {name} {code}' : 'Terminal - {name}',
+    }, stop);
+    const description = intl.formatMessage({
+      id: 'stop-page.description',
```
Comment (hannesj) [inline app/component/StopPageMeta.js]: terminal-page.description?

File: app/component/StopPageMeta.js [hunk 3]
```diff
@@ -29,6 +46,7 @@ export default Relay.createContainer(StopPageMeta, {
       fragment on Stop {
         name
         code
+        desc
```
Comment (hannesj) [inline app/component/StopPageMeta.js]: Is/should this be used somewhere?

File: app/component/StopPageMeta.js (hunk 2 above)
Comment (hannesj) [inline app/component/StopPageMeta.js]: terminal-page.description?

File: app/component/SummaryPage.js [hunk 4]
```diff
@@ -254,6 +254,8 @@ class SummaryPage extends React.Component {
 }
 
 
+export const customizableParameters = SummaryPage.customizableParameters;
```
Comment (hannesj) [inline app/component/SummaryPage.js]: is this used somewhere?

File: app/component/TopLevel.js [hunk 5]
```diff
@@ -53,7 +55,9 @@ class TopLevel extends React.Component {
 
   render() {
     configureMoment(this.context.intl.locale);
-    const metadata = meta(this.context.intl.locale);
+    const host = this.context.headers && this.context.headers.host;
```
Comment (hannesj) [inline app/component/TopLevel.js]: This needs to be checked that it works behind the digitransit-proxy

Comment (Macroz) [inline app/component/TopLevel.js]: Yes

File: app/config.hsl.js (hunk 1 above)
Comment (Macroz) [inline app/config.hsl.js]: Also confirmed OK in sprint planning.

**Pull request merged**